### PR TITLE
Regex for spacer detection fixed

### PR DESCRIPTION
**Pull request by Najsr**
Fixed regex for spacer detection

### CODE REVIEW

File: TS3QueryLib.Core.Framework/Common/Entities/SpacerInfo.cs [hunk 1]
```diff
@@ -30,14 +30,14 @@ public static SpacerInfo Parse(string channelName)
             if (channelName.IsNullOrTrimmedEmpty())
                 return null;
 
-            const string pattern = @"\[(?<Alignment>r|l|c|\*)spacer.*?\](?<VisibleName>.*)";
+            const string pattern = @"\[(?<Alignment>r|l|c|\\*)spacer.*?\](?<VisibleName>.*)";
```
Comment (Scordo) [inline TS3QueryLib.Core.Framework/Common/Entities/SpacerInfo.cs]: Why did you add a second backslash? the first one was an escape for the star (*).

Valid values are:
* [*spacer]---
* [rspacer]---
* [cspacer]---
* [lspacer]---

The regex you've made is not correct, because it allows:
* [rspacer]---
* [cspacer]---
* [lspacer]---
* [\spacer]---     (<--Wrong)
* [\\\\\\spacer]---     (<--Wrong)

Regards,
Scordo

**Comment by Najsr** [comment]
Oh yeah, by bad. I've changed it. Not it does not require an alignment char + if parent id is not 0 then it is not spacer. So this should make the detection better overall.

**Comment by Najsr** [comment]
Hey, I fixed the regex some time ago, could you look into it?
